### PR TITLE
Improved file structure

### PR DESCRIPTION
**Pull request by daraghmeehan**
Hi Jirka,

Do you think you could check my current file structure to see if it looks good?
I've tried to create clean separate modules for the core app, tests, and other resources/data.

### CODE REVIEW

File: app/main_module.bat [hunk 1]
```diff
@@ -1,4 +1,4 @@
 call conda activate universalearn
-python lang_anki_v2.py
+python main_module.py
 call conda deactivate
 pause
```
Comment (jirka-x1) [inline app/main_module.bat:4]: Always have a new-line at the end of files (this is because of UNIX cmd tools)

**Comment by jirka-x1** [comment]
The structure is ok.

Regarding other things, you need to add more docstrings and do use type hints for all parameters and return values. (no need to address that in this PR)

File: app/main_module.bat (hunk 1 above)
Comment (daraghmeehan) [reply app/main_module.bat:4]: Got it!

Comment (daraghmeehan) [reply app/main_module.bat:4]: I've fixed the new-line issue.

Yes I will do complete docstrings in a new branch!

**Comment by daraghmeehan** [comment]
Closing this. I addressed the .bat new-line issue in my current branch

**Pull request merged**